### PR TITLE
4612 save sig

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/presenter/actions/completeDocumentSigningAction.js [hunk 1]
```diff
@@ -4,19 +4,13 @@ import { state } from 'cerebral';
  * generates an action for completing document signing
  *
  * @param {object} providers the providers object
- * @param {string} providers.successMessage the success message to display in the success alert
+ * @param {string} providers.get the cerebral get function
```
Comment (matthopson) [inline web-client/src/presenter/actions/completeDocumentSigningAction.js:7]: missing `applicationContext`

File: web-client/src/presenter/actions/setCompleteDocumentSigningSuccessAlertAction.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ *
+ * @returns {object} the props with the message
+ */
+export const setCompleteDocumentSigningSuccessAlertAction = () => {
+  return {
+    alertSuccess: {
+      message: 'Signature added.',
+    },
+  };
+};
```
Comment (matthopson) [inline web-client/src/presenter/actions/setCompleteDocumentSigningSuccessAlertAction.js:12]: Test?

File: web-client/src/presenter/actions/setSignAndCompleteDocumentSigningSuccessAlertAction.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * @returns {object} the props with the message
+ */
+export const setSignAndCompleteDocumentSigningSuccessAlertAction = () => {
+  return {
+    alertSuccess: {
+      message:
+        'Your document has been successfully created and attached to this message',
+    },
+  };
+};
```
Comment (matthopson) [inline web-client/src/presenter/actions/setSignAndCompleteDocumentSigningSuccessAlertAction.js:13]: Test

File: web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { state } from 'cerebral';
+
+/**
+ * set the success message in props for successful document signing
+ *
+ * @returns {object} the props with the message
```
Comment (matthopson) [inline web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.js:8]: missing `get`

File: web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+export const setSuccessFromDocumentTitleAction = ({ get }) => {
+  const { documents } = get(state.caseDetail);
+  const documentId = get(state.documentId);
+  const order = documents.find(d => d.documentId === documentId);
+  return {
+    alertSuccess: {
+      message: `${order.documentTitle} updated.`,
+    },
+  };
+};
```
Comment (matthopson) [inline web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.js:19]: Test

File: web-client/src/presenter/sequences/saveDocumentSigningForMessageSequence.js [hunk 6]
```diff
@@ -1,23 +1,22 @@
...
 import { navigateToMessageDetailAction } from '../actions/navigateToMessageDetailAction';
 import { parallel } from 'cerebral';
 import { setAlertSuccessAction } from '../actions/setAlertSuccessAction';
 import { setDocumentDetailTabAction } from '../actions/setDocumentDetailTabAction';
 import { setDocumentIdAction } from '../actions/setDocumentIdAction';
 import { setSaveAlertsForNavigationAction } from '../actions/setSaveAlertsForNavigationAction';
+import { setSignAndCompleteDocumentSigningSuccessAlertAction } from '../actions/setSignAndCompleteDocumentSigningSuccessAlertAction';
 import { showProgressSequenceDecorator } from '../utilities/sequenceHelpers';
 
 export const saveDocumentSigningForMessageSequence = showProgressSequenceDecorator(
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/saveDocumentSigningForMessageSequence.js:14]: Are you able to reuse any of the code from `saveDocumentSigningSequence` here? My fear is that these sequences will start to diverge.

File: web-client/src/presenter/actions/computeFormDateAction.test.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+          month: '2',
+          year: '2002',
+        },
+      },
+    });
+
+    expect(result.output.computedDate).toEqual('2002-02-01');
+  });
+
+  it('should return null if month is not defined', async () => {
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/computeFormDateAction.test.js]: but in this case, `month` *is* defined. Want to rephrase?

**Review by kkoskelin — APPROVED** [review]
Would like to see a test description updated (see my note) and others fixed. after that, 👍

File: web-client/src/presenter/sequences/saveDocumentSigningForMessageSequence.js (hunk 6 above)
Comment (codyseibert) [reply web-client/src/presenter/sequences/saveDocumentSigningForMessageSequence.js:14]: it seems like the sequences call a couple of different actions throughout the sequence.  I'm not sure it's worth it.

**Review by matthopson — APPROVED** [review]

**Pull request merged**